tools/clang/test/CodeGenHLSL/declarations/structs/builtin_types_no_inheritance.hlsl

```diff
@@ -1,34 +1,33 @@
 // RUN: %dxc -T vs_6_0 -E main %s | FileCheck %s
 
 // CHECK: error: base 'vector' is marked 'final'
-// CHECK: error: base 'matrix' is marked 'final'
-// CHECK: error: base 'Texture3D' is marked 'final'
-// CHECK: error: base 'ByteAddressBuffer' is marked 'final'
-// CHECK: error: base 'StructuredBuffer' is marked 'final'
-// CHECK: error: base 'SamplerState' is marked 'final'
-// CHECK: error: base 'TriangleStream' is marked 'final'
-// CHECK: error: base 'InputPatch' is marked 'final'
-// CHECK: error: base 'OutputPatch' is marked 'final'
-// CHECK: error: base 'RayDesc' is marked 'final'
-// CHECK: error: base 'BuiltInTriangleIntersectionAttributes' is marked 'final'
-// CHECK: error: base 'RaytracingAccelerationStructure' is marked 'final'
-// CHECK: error: base 'GlobalRootSignature' is marked 'final'
-
 struct F2 : float2 {};
+// CHECK: error: base 'matrix' is marked 'final'
```

Comment:
This looks like it should be a Verifier test, rather than CodeGen.  Not that you're adding it here, just modifying it.

Reply:
True, I just changed it!